CDP4-SDK-Samples/SampleApp.cs

```diff
@@ -0,0 +1,506 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SampleApp.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2020 RHEA System S.A.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4.SDK.Samples
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Text;
+    using System.Threading.Tasks;
+    using CDP4Common.EngineeringModelData;
+    using CDP4Common.SiteDirectoryData;
+    using CDP4Common.Types;
+    using CDP4Dal;
+    using CDP4Dal.DAL;
+    using CDP4Dal.Operations;
+    using CDP4ServicesDal;
+    using NLog;
+    using NLog.Config;
+    using NLog.Targets;
+
+    public class SampleApp
+    {
+        private static ISession _session;
+        private static Credentials _credentials;
+        private static bool _isRunning = true;
+        private static Uri _uri;
+
+        // Commands
+        private const string Open = "open";
```

Comment:
for `private` fields we use camelCase even for `const`.

Reply:
Not for private const !
![image](https://user-images.githubusercontent.com/56023674/71962825-99caa100-31fa-11ea-86fe-9ae9dd899a99.png)